### PR TITLE
Add ping community script

### PR DESCRIPTION
**Pull request by cammellos**
Add a script to periodically publish on a community test channel.

### CODE REVIEW

**Review by Samyoul — APPROVED** [review]
LGTM

**Review by jakubgs — COMMENTED** [review]
No `make ping-community` target in main `Makefile`?

File: cmd/ping-community/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+### How to build
```
Comment (jakubgs) [inline cmd/ping-community/README.md:4]: This `README` doesn't actually describe what this command does.

File: cmd/ping-community/main.go [hunk 2]
```diff
@@ -0,0 +1,459 @@
...
+	fmt.Println("OS:", runtime.GOOS)
+	fmt.Printf("GOPATH=%s\n", os.Getenv("GOPATH"))
+	fmt.Printf("GOROOT=%s\n", runtime.GOROOT())
+
+	fmt.Println("Loaded Config: ", config)
+}
+
+func printUsage() {
+	usage := `
+Usage: statusd [options]
```
Comment (jakubgs) [inline cmd/ping-community/main.go]: This is just copy-pasted from other commands. Wouldn't it be good to update this and describe what this actually does?

**Pull request merged**